src/Migrations/Version20250310142443.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20250310142443 extends AbstractMigration
+{
+    #[\Override]
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    #[\Override]
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE administrateur CHANGE user_id user_id INT DEFAULT NULL');
+        $this->addSql('ALTER TABLE beneficiaire DROP FOREIGN KEY FK_B140D802B660D3F4');
+        $this->addSql('ALTER TABLE beneficiaire CHANGE user_id user_id INT DEFAULT NULL, CHANGE totalFileSize totalFileSize INT DEFAULT NULL');
```

Comment:
`CHANGE user_id user_id INT DEFAULT NULL` could be a problem.

Reply:
Actually user is defined in Subject parent entity as Nullable